### PR TITLE
Move loaders for css files to devDependencies

### PR DESCRIPTION
**Pull request by paulathevalley**
Since `.css` files are only used for the demo, move the loaders to `devDependencies` and remove them from the root webpack config.

### CODE REVIEW

**Comment by paulathevalley** [comment]
Remember how node v6 was failing (#96)? Now it’s passing, but node v4 is failing! 

https://travis-ci.org/FormidableLabs/component-playground/jobs/278014606#L558

```
21 09 2017 02:04:32.985:INFO [karma]: Karma v0.13.22 server started at http://localhost:9999/
21 09 2017 02:04:32.993:INFO [launcher]: Starting browser PhantomJS
21 09 2017 02:04:32.998:ERROR [launcher]: No binary for PhantomJS browser on your platform.
  Please, set "PHANTOMJS_BIN" env variable.
21 09 2017 02:04:33.000:ERROR [launcher]: [TypeError: Cannot read property 'stderr' of undefined]
TypeError: Cannot read property 'stderr' of undefined
    at Object._start (/home/travis/build/FormidableLabs/component-playground/node_modules/karma-phantomjs-launcher/index.js:82:18)
```

File: package.json [hunk 1]
```diff
@@ -67,45 +67,46 @@
...
     "karma-safari-launcher": "^0.1.1",
     "karma-sauce-launcher": "^0.2.14",
     "karma-spec-reporter": "0.0.20",
-    "karma-webpack": "^1.6.0",
+    "karma-webpack": "^1.7.0",
     "lodash": "^4.13.1",
     "mocha": "^2.2.5",
     "opener": "^1.4.1",
-    "phantomjs": "^1.9.17",
+    "phantomjs-prebuilt": "^2.1.15",
```
Comment (paulathevalley) [inline package.json]: Evidently `yarn` can build `phantomjs-prebuilt` but not `phantomjs`!

**Comment by paulathevalley** [comment]
Ready for review!

File: package.json (hunk 1 above)
Comment (ryan-roemer) [reply package.json]: `phantomjs` is deprecated. `phantomjs-prebuilt` is the modern version.

File: .travis.yml [hunk 2]
```diff
@@ -1,9 +1,11 @@
 language: node_js
 
 node_js:
-  - "4"
-# Disabling tests on node6 until #96 is resolved
-#- "6"
+  - 4
+  - 6
```
Comment (ryan-roemer) [inline .travis.yml]: Add `- 8` please.

File: package.json [hunk 3]
```diff
@@ -67,45 +67,46 @@
...
     "prop-types": "^15.5.10",
     "react": "^15.4.2",
     "react-addons-test-utils": "^15.4.2",
     "react-codemirror": "^1.0.0",
     "react-dom": "^15.4.2",
     "rimraf": "^2.4.0",
-    "style-loader": "~0.8.0",
-    "url-loader": "~0.5.5",
-    "webpack": "^1.10.0"
+    "webpack": "^1.12.3"
```
Comment (ryan-roemer) [inline package.json]: Can we ticket out upgrading to `webpack@3` later?

Comment (paulathevalley) [reply package.json]: Done: https://github.com/FormidableLabs/component-playground/issues/102

File: .travis.yml (hunk 2 above)
Comment (paulathevalley) [reply .travis.yml]: 👍  ✅

**Pull request merged**